packages/icons/src/resource-stack.svg

```diff
@@ -0,0 +1,9 @@
+<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Worth running these through an optimiser ((ImageOptim)[https://imageoptim.com/mac] is my favourite). A quick test shows a 1/3rd reduction in size.

Reply:
will do that now